lib/board.ex

```diff
@@ -1,31 +1,51 @@
 defmodule Board do
   defstruct [:board]
 
+  @row_count 10
+  @col_count 10
```

Comment:
The evolution of this concept has been cool to see! I'm wondering if this doesn't feel like game configuration/settings now. What do you think? Am I thinking too far ahead?

Reply:
Thank you, @Thomascountz! I've definitely been thinking about when/if we want to have different sized boards, and I've been making notes in my head of what things I'll need to change for that as I'm writing them. I was thinking that I would leave it for now until it becomes necessary, but maybe I should have started on that earlier?